doc/model.md

```diff
@@ -1,10 +1,12 @@
-# XGBoost Parameter Document
+# Model Document
```

Comment:
What is a "Model Document"? The title is very confusing. I can understand a file named `pkg/sql/codegen/xgboost/README.md` is a about the XGBoost codegen; however, I have no idea what will a `model.md` file is supposed to present.

Reply:
How about renaming it to "Model Parameter Document"?

Reply:
You are right, the name `model.md` is confusing. I will change it to `model_parameter.md`.